Source/common/SNTPrefixTree.cc

```diff
@@ -85,7 +85,7 @@ IOReturn SNTPrefixTree::AddPrefix(const char *prefix, uint64_t *node_count) {
       }
 
       // Is there enough room for the rest of the prefix?
-      if ((node_count_ + (len - i)) > max_nodes_) {
+      if ((node_count_ + (len - (size_t)i)) > max_nodes_) {
```

Comment:
Is there a reason for adding this cast?

Reply:
This is reviving an old branch, I'm not sure what that was for. Reverted.